doxygen/plan.sh

```diff
@@ -1,12 +1,16 @@
 pkg_name=doxygen
 pkg_origin=core
-pkg_version=1.8.11
+pkg_version_major="1"
+pkg_version_minor="8"
+pkg_version_bugfix="15"
+pkg_version="${pkg_version_major}.${pkg_version_minor}.${pkg_version_bugfix}"
 pkg_license=('GPL-2.0')
 pkg_description="Generate documentation for several programming languages"
-pkg_upstream_url=http://www.doxygen.org/
+pkg_upstream_url="http://www.doxygen.nl/"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
-pkg_source="https://ftp.stack.nl/pub/users/dimitri/${pkg_name}-${pkg_version}.src.tar.gz"
-pkg_shasum=65d08b46e48bd97186aef562dc366681045b119e00f83c5b61d05d37ea154049
+pkg_source="https://github.com/doxygen/doxygen/archive/Release_${pkg_version_major}_${pkg_version_minor}_${pkg_version_bugfix}.tar.gz"
```

Comment:
doxygen offers a download site at `"http://doxygen.nl/files/${pkg_name}-${pkg_version}.src.tar.gz"`  If you update to use that location, the `pkg_version_major` `pkg_version_minor` ``pkg_version_bugfix`  and `pkg_dirname` variables aren't needed. 😄